Minio.Functional.Tests/FunctionalTest.cs

```diff
@@ -1457,11 +1492,14 @@ internal async static Task CopyObject_Test8(MinioClient minio)
                 {
                     await minio.PutObjectAsync(bucketName,
                                             objectName,
-                                            filestream, filestream.Length, metaData:new Dictionary<string, string>{{"Orig", "orig-val with  spaces"}});
+                                            filestream, filestream.Length, "application/octet-stream", metaData:new Dictionary<string, string>{{"Orig", "orig-val with  spaces"}});
                 }
-                ObjectStat stats = await minio.StatObjectAsync(bucketName, objectName);
+                StatObjectArgs statObjectArgs = new StatObjectArgs()
+                                                        .WithBucket(bucketName)
+                                                        .WithObject(objectName);
+                ObjectStat stats = await minio.StatObjectAsync(statObjectArgs);
 
-                Assert.IsTrue(stats.MetaData["Orig"] != null) ;
+                Assert.IsTrue(stats.MetaData["orig"] != null) ;
```

Comment:
why is this changing - the metadata key is case sensitive

Reply:
Metadata dictionary with comparer IgnoreCase. Code updated.